feat: CPU 병렬처리

yeoV opened the pull request:
CPU연산 시, 사용 가능한 core수 만큼
병렬처리

resolve: #6

설명
사용가능한 core 수 출력되고 사용되게 해놨는데 한번 테스트 해줘욥.,
그 측정시간 체크하려면 주석처리 된거에서, time import하구
t 변수랑 밑에 total 시간 출력하는 print문 주석 풀고 실행하면 나왓

yeoV committed:
feat: CPU 병렬처리
CPU연산 시, 사용 가능한 core수 만큼
병렬처리

resolve: #6

always0ne reviewed (approved):
totaltime 이거 로그남기는용으로 주석해제해도 좋을듯
requirement 이거 지금 파일 두갠데 하나 지우는게 좋을듯

yeoV committed:
feat: CPU처리 및 경과시간 로그
CPU연산 시, 사용 가능한 core수 만큼
병렬처리.
+경과시간 log 남기기 추가.
- requirement 중복 삭제
resolve: #6
